community/sway/usr/share/sway/scripts/wob.sh

```diff
@@ -12,17 +13,15 @@ is_running_on_this_screen() {
     return 1
 }
 
-new_value=$1 # null or a percent; no checking!!
-
-color=${2:-16a085}
+new_value=$3 # null or a percent; no checking!!
 
 wob_pipe=~/.cache/$( basename $SWAYSOCK ).wob
 
 [[ -p $wob_pipe ]] || mkfifo $wob_pipe
 
 # wob does not appear in $(swaymsg -t get_msg), so:
 is_running_on_this_screen wob || {
-    tail -f $wob_pipe | wob --border-color "#FF${color}" --bar-color "#FF${color}" --background-color '#FF000000' --anchor top --anchor center --margin 20 &
+    tail -f $wob_pipe | wob --border-color $1 --bar-color $1 --background-color $2 --anchor top --anchor center --margin 20 &
```

Comment:
```suggestion
    tail -f $wob_pipe | wob --border-color "$1" --bar-color "$1" --background-color "$2" --anchor top --anchor center --margin 20 &
```

Reply:
otherwise "#" would comment stuff out

Reply:
I believe it doesn't work like that in the bash scripts, the variables (arguments in this case) should get escaped. Definitely works for me without quotes

Reply:
ya, I tried and it seems to work nicely. nvm